pkg/virt-operator/creation/components/crds.go

```diff
@@ -314,7 +314,7 @@ func NewPrometheusRuleSpec(ns string) *promv1.PrometheusRuleSpec {
 					{
 						Record: "num_of_running_virt_api_servers",
 						Expr: intstr.FromString(
-							fmt.Sprintf("sum(up{namespace='%s', pod='virt-api-.*'})", ns),
+							fmt.Sprintf("sum(up{namespace='%s', pod=~'virt-api-.*'})", ns),
```

Comment:
if there's a way to sum by pods with labels, that might be more accurate. the label would be `kubevirt.io: virt-api`. that same convention is used for all the other components as well.

Reply:
unfortunately the pod labels are not exposed in the `up` metric, so we can't filter by it.